### ISSUE TITLE
What is the score of trainer.predict()?

### ISSUE THREAD

**Issue by Yuukp** [huggingface/transformers#10960]
I want to know the meaning of output of trainer.predict().

example:
`PredictionOutput(predictions=array([[-2.2704859,  2.442343 ]], dtype=float32), label_ids=array([1]), metrics={'eval_loss': 0.008939245715737343, 'eval_runtime': 0.0215, 'eval_samples_per_second': 46.56})`

What is this score? -> predictions=array([[-2.2704859,  2.442343 ]]

I use it for Sequence Classification.

**Comment by frankhart2018** [comment]
These are the logits from your model, check:- https://github.com/huggingface/transformers/blob/8780caa388c7b2aa937454ed96bcdd3f097f851d/src/transformers/trainer.py#L1852

**Comment by Yuukp** [comment]
I see! Thank you so much!!
